### ISSUE TITLE
Does not install 

### ISSUE THREAD

**Issue by Moon8008** [ZaneYork/SMAPI-Android-Installer#245]
When I try to install the app it tells me that it was not installed :(

**Comment by Gahahjkif567** [comment]
I installed the smapi installer everything works till i click the install button in the end it says the app wasn't installed, please help me

**Comment by Gahahjkif567** [comment]
> When I try to install the app it tells me that it was not installed :(

My exact problem

**Comment by lissyloo23** [comment]
Yup doesn't work for me either which is funny because I couldn't get it to work on my laptop either haha guess I'm just gonna give up 😂

**Comment by Kayalanrij** [comment]
Same here. I'm playing Stardew Valley on an Android 10 tablet and the SMAPI installation process apparently works fine: unpacking, patching, signing, then I'm asked if I want to install the app and tap "yes" - but in the last popup of the installation process where I expect "installation successful" or something like that it says "app was not installed". Without any error message or hint about what went wrong. I gave the SMAPI app the permission to install from an unknown source and the files permission. I tried restarting my device and tried to install SMAPI several times, and I get this error message every time. Does anyone have a clue how to fix this? Oh, and I'm using v3.7.6.9 of SMAPI since this was recommended for Android devices.

**Comment by Kayalanrij** [comment]
I found an information where to look for the SMAPI logs [here](https://smapi.io/log), but there's no ErrorLogs folder on my device. Where does the installer log it's errors?
![IMG_20230603_161839](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/135431739/940c6868-9908-40d1-92cc-836965e35327)

**Comment by Kayalanrij** [comment]
I tried the manual install next as described [here](https://github.com/ZaneYork/SMAPI-Android-Installer/issues/301) and was actually able to install the SMAPI beta version 3.18.2.2. But it crashes during game startup. 😥
![Screenshot_2023-06-03-17-18-26-033](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/135431739/5be69a1f-a17c-4ed8-a5e4-7a4c9a6b9f0b)
